nifi-commons/nifi-sensitive-property-provider/src/main/java/org/apache/nifi/properties/SensitivePropertyProviderFactory.java

```diff
@@ -33,4 +33,14 @@
      */
     Collection<SensitivePropertyProvider> getSupportedSensitivePropertyProviders();
 
+    /**
+     * Given the &lt;identifier&gt; value providing a group context for an XML configuration file property,
+     * returns the matching property context, or the default context if no mappings match.
+     * @param groupIdentifier The value of &lt;identifier&gt; from the enclosing XML block of a property in
+     *                        one of the XML configuration files.
+     * @param propertyName An XML property name
```

Comment:
The description references to XML seem specific to the implementation, can this be adjusted?